pylint/checkers/misc.py

```diff
@@ -134,45 +133,16 @@ def process_tokens(self, tokens: list[tokenize.TokenInfo]) -> None:
         """Inspect the source to find fixme problems."""
         if not self.linter.config.notes:
             return
-        comments = (
-            token_info for token_info in tokens if token_info.type == tokenize.COMMENT
-        )
-        for comment in comments:
-            comment_text = comment.string[1:].lstrip()  # trim '#' and white-spaces
-
-            # handle pylint disable clauses
-            disable_option_match = OPTION_PO.search(comment_text)
```

Comment:
Yeah, we were seeing if there is a `# pylint: disable=fixme` on the same line as the `fixme` comment. However, we have other code that does this for us and it doesn't allow using `disable-next`.
Just removing is fine.

Reply:
I don't think it actually does anything other than disallow `disable`. `PragmaParserError` probably catches all other issues.

The fact that all of our `enable` tests work without this shows that it is completely unnecessary 😄

Reply:
Sorry for asking but this cleanup looked too good to be true 😄 I'm just amazed at the number of time we just remove a bunch of code to fix a bug. It's been happening over and over.

Reply:
😄 

I also was quite surprised when I found this code. At first I though for some reason the `fixme` checker was responsible for parsing all `pragma's` but then I remembered that I actually refactored that code myself and made sure that it will always be `PyLinter `... 😅

Reply:
Having refactored half the codebase has its perks 😄